src/constants.h

```diff
@@ -24,7 +24,7 @@
 #define MAX_NUM_BASIN_SURFACES 12 // the maximum number of basin surfaces
 #define MAX_NUM_1D_VELOCITY_MODELS 1 // the maximum number of 1D velocity models
 #define MAX_NUM_BASIN_BOUNDARIES 3 // the maximum number of boundaries per velocity model
-#define MAX_NUM_GEN_MULTI_PROFILES 27000 // the maximum number of profiles able to be generated by GEN_MULTI_PROFILES_CALL
+#define MAX_NUM_GEN_MULTI_PROFILES 300000 // the maximum number of profiles able to be generated by GEN_MULTI_PROFILES_CALL
```

Comment:
I assume this is for creating a static array, does it have a noticeable impact on ram use?

Reply:
Nope, Larger than this did seem to start running away though.

Reply:
Looks like it's gone from 12.5Mb to ~125Mb ram use, so looks ok for now